File: classwork/vowel.js
let sentence = "I am a boy i leav in ginns"
	sentence = sentence.toUpperCase()
const vowel = "AEIOU"
let totalvowel = 0

for(var count = 0; count < sentence.length;count++){
	
	for(var counter = 0; counter < vowel.length;counter++){

		if(sentence.charAt(count) == vowel.charAt(counter)) totalvowel++
	}
}

console.log(totalvowel)

File: classwork/Vowel.java
import java.util.Scanner;
public class Vowel {
	public static void main (String [] args){
	Scanner input = new Scanner(System.in);
	String vowel = "AEIOU";
	 // checkVowel("AEIOU");
	System.out.println("Enter The Sentence");
	String sentence = input.nextLine();
	sentence = sentence.toUpperCase();

	int totalVowel = 0;
	int count = 0;
	do{
		for(int counter = 0; counter < 5; counter++){
			if(sentence.charAt(count) == vowel.charAt(counter)) totalVowel++;
		
		}
		count++;

	}while(count < sentence.length());

	System.out.println("The total vowel is  " + totalVowel);
	}


}

